src/DIRAC/WorkloadManagementSystem/Agent/OptimizerModule.py

```diff
@@ -48,17 +48,12 @@ def initialize(self, jobDB=None, logDB=None):
         if not self.jobDB.isValid():
             dExit(1)
 
-        useESForJobParametersFlag = Operations().getValue("/Services/JobMonitoring/useESForJobParametersFlag", False)
-        if useESForJobParametersFlag:
-            try:
-                result = ObjectLoader().loadObject(
-                    "WorkloadManagementSystem.DB.ElasticJobParametersDB", "ElasticJobParametersDB"
-                )
-                if not result["OK"]:
-                    return result
-                self.elasticJobParametersDB = result["Value"]()
-            except RuntimeError as excp:
-                return S_ERROR(f"Can't connect to DB: {excp}")
+        result = ObjectLoader().loadObject(
+            "WorkloadManagementSystem.DB.ElasticJobParametersDB", "ElasticJobParametersDB"
+        )
+        if not result["OK"]:
+            return result
+        self.elasticJobParametersDB = result["Value"]()
```

Comment:
Is that try/except for RuntimeError that was there before, really no longer needed?

Reply:
I think so.

Reply:
ElasticJobParametersDB still throws RuntimeError
e.g.: 
https://github.com/DIRACGrid/DIRAC/pull/7292/files#diff-343b190022086de54ccf30f9a1ae0e7b1c5f5b312f95f48e9c91abf17f210dbaR43

Reply:
No, the throwing comes from calling the c'tor of ElasitcJobParametersDB in this line here, doesn't it?
```python
        self.elasticJobParametersDB = result["Value"]()
                                                     ^^
```

Reply:
Correct. But, anyway, this is `initialize()` and it's OK to see the exception, if it happens.